package.json

```
@@ -12,7 +12,8 @@
     "test": "jest && npm run test-types",
     "test-types": "tsc --noEmit -p ./tests/tsconfig.tests.json",
     "lint": "eslint ./src --ext .ts",
-    "format": "prettier --write 'src/**/*.ts?(x)' && npm run lint -- --fix",
+    "format": "prettier --write . && npm run lint -- --fix",
+    "format-check": "prettier --check .",
```

Lets utilize prettier and eslint's cacheing options 

- https://prettier.io/docs/en/cli#--cache
- https://eslint.org/docs/latest/use/command-line-interface#caching

```suggestion
    "lint": "eslint ./src --ext .ts --cache --cache-location 'node_modules/.cache/eslintcache",
    "format": "prettier --write --cache --cache-location='node_modules/.cache/prettiercache' . && npm run lint -- --fix",
    "format-check": "prettier --check --cache --cache-location='node_modules/.cache/prettiercache' .",
```

---

Getting this warning from Prettier:

```
[warn] Ignored unknown option --cache.
[warn] Ignored unknown option --cache-location=node_modules/.cache/prettiercache.
```